build.gradle

```diff
@@ -52,7 +56,7 @@ allprojects {
         }
     }
 
-    tasks.withType(Test) {
+    tasks.withType(Test).configureEach { test ->
```

Comment:
Out of curiosity, what's the difference?

Reply:
I found it here. https://blog.gradle.org/preview-avoiding-task-configuration-time
`configureEach` configures more or less of the build depending on what is required for execution. 😄

Reply:
Thanks for an explanation. Very useful. :bow: